src/main/java/com/playmonumenta/networkchat/utils/ReplacerWithEscape.java

```diff
@@ -110,6 +114,13 @@ public String apply(MatchResult matchResult) {
 
 		String result = builder.toString();
 		debugMessage("result: ", result);
+
+		if (mHasPlaceholder) {
+			result = PlaceholderAPI.setPlaceholders((mSender instanceof Player player) ? player : null, result);
+		}
+
+		result = Matcher.quoteReplacement(result);
```

Comment:
I'm pretty sure quoting the replacement was just for debugging - everything else looks good to go!

Reply:
see [discord](https://discord.com/channels/186225508562763776/343478653449142273/1173492167482814544)!